.github/workflows/gh-pages.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   deploy:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Please do not use `ubuntu-latest`, instead specify it to `ubuntu-20.04`.

Reply:
Changed as suggested. Do you think it will work now?

Reply:
According to annotation - yes, it should :)